oi/type_graph/TypeIdentifier.cpp

```diff
@@ -20,6 +20,15 @@
 
 namespace type_graph {
 
+// TODO:
+// - read these from a TOML file
+// - don't require specifying ctype and header
```

Comment:
do you not need to specify the headers here? you could use "cstddef" in that case which is always included, but it looks to me like you do need the headers in weird edge cases

Reply:
Each of these dummy containers belongs to a real container and will only appear when that container is used. That means their header files will already be included by the real container.

Actually, typing this out has made me think that maybe "dummy container" should be a field in the container TOML of each real container that these belong to

Reply:
Oh, except std::allocator, which doesn't belong to any single container, but is used by all of them